requirements.txt

```diff
@@ -20,7 +20,7 @@ h5py==3.6.0
 idna==3.2
 keras==2.7.0
```

Comment:
i think you can remove these?  keras*

Reply:
that's my guess.  let's retrigger tests and see?

Reply:
huh. now they fail with the `StructuredValue` failure. Not _quite_ the same as https://github.com/google/ml-compiler-opt/runs/7932290113?check_suite_focus=true, though.